src/actions/index.js

```diff
@@ -15,23 +15,20 @@ export const setFavorites = (favorites, user_id) => ({
   user_id
 })
 
-// Users key 
-
+// User key 
 export const setUser = (user) => ({
   type: 'SET_USER',
   user,
 })
 
-// LoginPrompt key
-
+// Login key
 export const setShouldPromptLogin = (bool) => ({
```

Comment:
oh no, it lives!!!

Reply:
🤣 This will definitely go before Wednesday!